content/docs/introduction/faq.md

```diff
@@ -35,7 +35,8 @@ on the Robust Perception blog to get started.
 
 ### What language is Prometheus written in?
 
-Most Prometheus components are written in Go. Some are also written in Java and Ruby.
+Most Prometheus components are written in Go. Some are also written in Java,
+Python and Ruby.
```

Comment:
I suggest using Oxford comma here. And perhaps s/and/or/ (because no component is written in multiple languages at the same time).